.editorconfig

```diff
@@ -0,0 +1,18 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
+ij_typescript_use_double_quotes = false
+max_line_length = 140
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
I'm curious, what is the reason to not trim trailing whitespace in markdown files?

Reply:
Well spotted!
<img src="https://github.com/user-attachments/assets/2dea8d9e-3621-4e58-b998-e007261f871b" width="300" />
I've quickly checked this in another project. And we can definitely use `trim_trailing_whitespace = true` here. I'll change it.